test/controllers/content_items_controller_test.rb

```diff
@@ -119,14 +119,13 @@ class ContentItemsControllerTest < ActionController::TestCase
 
     content_store_has_item(content_item['base_path'], content_item)
 
-    with_variant EducationNavigation: "A" do
+    with_variant EducationNavigation: "A", assert_meta_tag: false do
```

Comment:
Why does the A variant have `assert_meta_tag: false`? Shouldn't the meta tag be added to both variants?